math/power/powvialogarithm.go

```diff
@@ -0,0 +1,23 @@
+// powvialogarithm.go
+// description: Powers in terms of logarithms
+// details:
+// implementation of exponentiation using exponent and logarithm, without using loops  - [Powers via logarithms wiki](https://en.wikipedia.org/wiki/Exponentiation#Powers_via_logarithms)
+// author(s) [red_byte](https://github.com/i-redbyte)
+// see powvialogarithm_test.go
+
+package power
+
+import (
+	"math"
+)
+
+func PowUseLog(a int, b int) int {
```

Comment:
shouldn't you keep the arguments `float64`? The `math.Log` and `math.Exp` use float64 so all the type casting you use will not be necessary.

```suggestion
func PowUseLog(a float64, b float64) float64 {
```

Reply:
Done